### PR TITLE
Approximate error display of REPL.

### PR DESCRIPTION
**Pull request by tpapp**
Follow up to #257, uses showerror and the output channel.

### CODE REVIEW

**Comment by tpapp** [comment]
This is a hack, because technically the API is not exposed and it may change. (cf [discussion](https://discourse.julialang.org/t/capture-error-from-include-string/121385)). But the error display resembles the REPL more closely.

File: src/Literate.jl [hunk 1]
```diff
@@ -936,15 +936,24 @@ function execute_block(sb::Module, block::String; inputfile::String, fake_source
...
-             ```
-             """)
+    if c.error
+        if continue_on_error
+            err = c.value
+            if err isa LoadError # include_string may wrap error in LoadError
+                err = err.error
+            end
+            error_output = "ERROR: " * sprint(showerror, err)
+            return nothing, error_output, disp.data
```
Comment (fredrikekre) [inline src/Literate.jl:946]: Probably this should be appended to `c.output` in case there are stderr/stdin output before the block throws an error.

**Comment by fredrikekre** [comment]
Thanks

**Comment by fredrikekre** [comment]
https://github.com/JuliaRegistries/General/pull/117528

**Pull request merged**